### PR TITLE
[Examples] remove examples from workflow and create a solution for examples.

### PR DESCRIPTION
**Pull request by Jim8y**
Closes https://github.com/neo-project/neo-devpack-dotnet/issues/1027

### CODE REVIEW

**Comment by Jim8y** [comment]
because examples will create artifacts for every example, it takes a long time, thus need to remove from the workflow. we can run it locally.

**Comment by cschuchardt88** [comment]
@Jim8y What is the problem with using
---
image

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]
Please don't change the formatting...

File: examples/Example.SmartContract.ContractCall.UnitTests/Example.SmartContract.ContractCall.UnitTests.csproj [hunk 1]
```diff
@@ -1,34 +1,34 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
+    <PropertyGroup>
```
Comment (shargon) [inline examples/Example.SmartContract.ContractCall.UnitTests/Example.SmartContract.ContractCall.UnitTests.csproj:3]: Agree with @cschuchardt88 and the format

**Review by superboyiii — APPROVED** [review]

**Comment by superboyiii** [comment]
Looks good on my local environment.

**Review by vncoelho — APPROVED** [review]

**Review by shargon — COMMENTED** [review]
Good change

**Pull request merged**